src/blocks/HeaderBlockBrandPivot/MenuItem.tsx

```diff
@@ -72,6 +82,24 @@ const MenuLink = styled('a')({
     paddingLeft: '1.5rem',
   },
 })
+
+const MenuGroupLabel = styled('span')({
+  display: 'block',
+  color: colorsV3.gray500,
+  textTransform: 'uppercase',
+  fontSize: '0.8rem',
+  lineHeight: '1rem',
+  whiteSpace: 'nowrap',
+  padding: 0,
+  paddingBottom: '1rem',
+
+  [TABLET_BP_DOWN]: {
+    display: 'inline-block',
+    padding: '0.5rem 0.5rem 0.5rem 1.5rem',
```

Comment:
I think you can remove the padding top here, it looks a bit tighter in the design 👍